### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jipgago**
CRUD 구현 완료 
Redis 구현시작

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,4 +4,7 @@ WORKDIR /app
 
 COPY . /app
 
-CMD ["./gradlew", "bootRun"]
\ No newline at end of file
+CMD ["./gradlew", "bootRun"]
+
+# Lombok 설치
+RUN java -jar lombok.jar install /alog-service-release
```
Comment (vivi108) [inline Dockerfile:10]: Why the command is needed? gradle package manager doesn't work?

Comment (jipgago) [reply Dockerfile:10]: Other packages work well, but Lombok has not run.

File: src/main/java/kea/alog/release/config/RedisConfig.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory;
+import org.springframework.data.redis.repository.configuration.EnableRedisRepositories;
+
+@EnableRedisRepositories
+@Configuration
+public class RedisConfig {
+    
+    private int port = 6379;
+    
+    private String host = "127.0.0.1";
```
Comment (vivi108) [inline src/main/java/kea/alog/release/config/RedisConfig.java]: let host be in docker-compose.yml, just like env.

File: src/main/java/kea/alog/release/config/RedisConfig.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory;
+import org.springframework.data.redis.repository.configuration.EnableRedisRepositories;
+
+@EnableRedisRepositories
+@Configuration
+public class RedisConfig {
+    
+    private int port = 6379;
```
Comment (vivi108) [inline src/main/java/kea/alog/release/config/RedisConfig.java]: let host be in docker-compose.yml, just like env. do not "hard-coding"

File: src/main/java/kea/alog/release/service/TagService.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    @Transactional
+    public void deleteTag(Long tagId){
+        Optional<Tag> optTag = tagRepository.findById(tagId);
+        if(optTag.isPresent()){
+            tagRepository.delete(optTag.get());
+        }
+    }
+
+    public TagPageingDTO getAllList(Long currentPage){
```
Comment (vivi108) [inline src/main/java/kea/alog/release/service/TagService.java:57]: @Transactional is missing

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    hibernate:
+      ddl-auto: update
+  redis: 
+    lettuce:
+      pool: 
+        max-active: 10
+        max-idle: 10
+        min-idle: 2
+      port: 6379
+      host: 127.0.0.1
```
Comment (vivi108) [inline src/main/resources/application.yml]: You've already define the port and host above, just like hard-coding-way
Why did you make env here also?
How about use @Value in code when you use envs?
And this should be define also in docker-compose.yml

**Review by vivi108 — CHANGES_REQUESTED** [review]

**Review by vivi108 — APPROVED** [review]
LGTM

**Pull request merged**